extensions/vertx/deployment/src/main/java/io/quarkus/vertx/deployment/VertxProcessor.java

```diff
@@ -51,6 +52,11 @@
     @Inject
     BuildProducer<ReflectiveClassBuildItem> reflectiveClass;
 
+    @BuildStep(providesCapabilities = Capabilities.RESTEASY_JSON_EXTENSION)
```

Comment:
It's not used for anything else?

Reply:
Which part do you mean? Sorry for my thick headedness :)

Reply:
The `RESTEASY_JSON_EXTENSION` Capability is used in `jackson` and `jsonb` extension to indicate `json` be support in `ResteasyCommonProcessor`. It was added for that case IIRC.

Reply:
Yes it already exists which is I guess what @gsmet was asking :)

Reply:
Well, I'm wondering if it's a good idea to reuse that one or if we should add a VERTX capability and alter the condition in the processor itself.

This capability was used to basically say "we have an extension to provide proper JSON support for RESTEasy" and this one is not exactly saying that. It's saying "Vert.x provides some sort of JSON support for its own types".

Maybe I'm overthinking it, I don't know.

Reply:
I think we are OK with that @machi1990 added for the time being. My view is that if we need something more complicated later, we can always improve things

Reply:
+1 @geoand 

WDYT @gsmet ?

Reply:
I think I agree with myself but let's do it your way and I'll keep my "I told you so" for later :).

Reply:
It wouldn't be the first time 😁

Reply:
> It wouldn't be the first time 😁

ha ha indeed :-)